code/ui/blocks/src/components/ArgsTable/ArgsTable.tsx

```diff
@@ -104,91 +103,58 @@ export const TableWrapper = styled.table<{
         },
       },
 
-      // Table "block" styling
-      // Emphasize tbody's background and set borderRadius
-      // Calling out because styling tables is finicky
-
       // Makes border alignment consistent w/other DocBlocks
       marginLeft: inAddonPanel ? 0 : 1,
       marginRight: inAddonPanel ? 0 : 1,
 
-      [`tr:first-child`]: {
-        [`td:first-child, th:first-child`]: {
-          borderTopLeftRadius: inAddonPanel ? 0 : theme.appBorderRadius,
-        },
-        [`td:last-child, th:last-child`]: {
-          borderTopRightRadius: inAddonPanel ? 0 : theme.appBorderRadius,
-        },
-      },
-
-      [`tr:last-child`]: {
-        [`td:first-child, th:first-child`]: {
-          borderBottomLeftRadius: inAddonPanel ? 0 : theme.appBorderRadius,
-        },
-        [`td:last-child, th:last-child`]: {
-          borderBottomRightRadius: inAddonPanel ? 0 : theme.appBorderRadius,
-        },
-      },
-
       tbody: {
-        // slightly different than the other DocBlock shadows to account for table styling gymnastics
-        boxShadow:
-          !inAddonPanel &&
-          (theme.base === 'light'
-            ? `rgba(0, 0, 0, 0.10) 0 1px 3px 1px,
-          ${transparentize(0.035, theme.appBorderColor)} 0 0 0 1px`
-            : `rgba(0, 0, 0, 0.20) 0 2px 5px 1px,
-          ${opacify(0.05, theme.appBorderColor)} 0 0 0 1px`),
-        borderRadius: theme.appBorderRadius,
-
-        // for safari only
-        // CSS hack courtesy of https://stackoverflow.com/questions/16348489/is-there-a-css-hack-for-safari-only-not-chrome
-        '@media not all and (min-resolution:.001dpcm)': {
-          '@supports (-webkit-appearance:none)': {
-            borderWidth: 1,
-            borderStyle: 'solid',
-            ...(inAddonPanel && {
-              borderColor: 'transparent',
-            }),
-
-            ...(!inAddonPanel && {
-              borderColor:
+        // Safari doesn't love shadows on tbody so we need to use a shadow filter. In order to do this,
+        // the table cells all need to be solid so they have a background color applied.
+        // I wasn't sure what kinds of content go in these tables so I was extra specific with selectors
+        // to avoid unexpected surprises.
+        ...(inAddonPanel
+          ? null
+          : {
+              filter:
                 theme.base === 'light'
-                  ? transparentize(0.035, theme.appBorderColor)
-                  : opacify(0.05, theme.appBorderColor),
+                  ? `drop-shadow(0px 1px 3px rgba(0, 0, 0, 0.10))`
```

Comment:
Do we still want to use `appBorderColor`?

Reply:
No. It isn't used on the canvas drop shadows so I modified this to match the canvas shadow styles.